src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts

```diff
@@ -310,7 +310,6 @@ export interface FacetDetails {
 }
 
 export interface PreparationConfigCC {
-    documentGroupId: number;
```

Comment:
I think that removing this is a breaking change.

Reply:
I agree with Louis, This is a mandatory config being removed from the interface :/ You should indicate that it is indeed breaking!

Reply:
@louis-bompart Thanks for the suggestion. I marked this pr as breaking.